engine/src/main/java/org/terasology/rendering/dag/nodes/FinalPostProcessingNode.java

```diff
@@ -105,14 +166,12 @@ public void process() {
     @Override
     public void propertyChange(PropertyChangeEvent event) {
         // This method is only called when oldValue != newValue.
-        if (!renderingDebugConfig.isEnabled()) {
-            removeDesiredStateChange(enableDebugMaterial);
-            addDesiredStateChange(enablePostMaterial);
-        } else {
-            removeDesiredStateChange(enablePostMaterial);
-            addDesiredStateChange(enableDebugMaterial);
-        }
+        if (event.getPropertyName().equals(RenderingConfig.FILM_GRAIN)) {
+            isFilmGrainEnabled = renderingConfig.isFilmGrain();
+        } else if (event.getPropertyName().equals(RenderingConfig.MOTION_BLUR)) {
+            isMotionBlurEnabled = renderingConfig.isMotionBlur();
+        } // else: no other cases are possible - see subscribe operations in initialize().
 
-        worldRenderer.requestTaskListRefresh();
+        // TODO: Do we have to monitor BlurIntensity?
```

Comment:
Sounds like a reasonable thing to do. Confusingly, this can be thought as related to the motion blur. This isn't the case it seems, as the BlurIntensity seems to be uniquely related to DoF instead.